### DISCUSSION TITLE
"The server is not reachable on Port 443." - Port 443 is OPEN

### DISCUSSION THREAD

**Discussion by axrusar** [nextcloud/all-in-one#982] [Questions] [answered]
I am using the AIO docker setup on an Ubuntu 20.04 VM server in vanilla state, no firewall, hostname and hosts setup correctly with a subdomain (cloud.mydomain.com) that points to the server's public IP. (no DNS propagation issues, it was setup 4 days ago already)
Portchecker.co shows port 443 is OPEN and reachable, but the nextcloud AIO first step where you have to input the domain, keeps telling me port 443 is not reachable.

I have a snapshot of this specific VM, with the clean, virgin Ubuntu server ready to run the docker command as per the instructions:

`sudo docker run -it \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
nextcloud/all-in-one:latest`

Once the containers are downloaded and launched, i am heading to the LAN ip of the server:8080 and login in with the multiple words password, then I try to input the domain and i keep getting the error.

To CONFIRM that there is no network issue, I did the following test:
Instead of running the docker setup for AIO, i installed apache on THE SAME vm i am using to setup AIO, then i installed certbot, and i was able to get a certificate from lets encrypt using the same domain name i am trying for the AIO setup.
This tells me there is no issues with port 443, and i am able to open the apache default page with the padlock from a remote network using the domain name mentioned.

Screenshots:
![aio-443](https://user-images.githubusercontent.com/110247477/181794326-907ce6b7-040d-4447-a688-dac66d8f0358.png)
![aio2](https://user-images.githubusercontent.com/110247477/181794521-dcdd61d5-dc13-4b6c-b550-bc927b27856a.png)

Any ideas what could be wrong or where to check?
I actually run the exact same setup at another location a few weeks ago and i had no issues, so i am puzzled here.
Thank you

**Comment by szaimen** [comment, marked as answer]
See https://help.nextcloud.com/t/nextcloud-aio-install-fails-to-find-open-port-443-open/142867/7
